examples/specs/window_running_average.vl.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A line graph showing how the average rating for a movie has changed over the years.",
+  "data": {"url": "data/movies.json"},
+  "width": 500,
+  "transform": [
+    {
+      "window": [{
+        "op": "mean",
+        "field": "IMDB_Rating",
+        "as": "AverageRating"
+      }],
+      "frame": [
+        null,
+        0
+      ]
+    },
+    {"filter": "datum.AverageRating != null"},
+    {"filter": "datum.AverageRating > 0"}
+  ],
+  "layer": [
```

Comment:
Why using layer with only one layer?  

A better plot would be showing both raw data and moving average.